presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergSmoke.java

```diff
@@ -53,7 +53,10 @@ protected QueryRunner createQueryRunner()
     @Test
     public void testTimestamp()
     {
-        // TODO
+        assertUpdate("CREATE TABLE test_timestamp (x timestamp)");
+        assertUpdate("INSERT INTO test_timestamp VALUES (timestamp '2017-05-01 10:12:34')", 1);
+        assertQuery("SELECT * FROM test_timestamp", "SELECT CAST('2017-05-01 10:12:34' AS TIMESTAMP)");
```

Comment:
any test to cover the time with zone?

Reply:
The timestamp with time zone will require changes in the ORC/parquet readers underneath. So I remove the timestamp with time zone support temporarily for now.